### ISSUE TITLE
conda: command not found

### ISSUE THREAD

**Issue by gabrielrajkovic** [biostars/biostar-handbook-issues#28]
Hi, I'm running bash on windows 10 and I followed the instructions to download miniconda. Then when I close the terminal, open it back up and type conda it says:
conda: command not found

I have a suspicion its not located in the right place but I'm not sure where it should be located. 
Thanks,
Gabe

**Comment by ialbert** [comment]
did you install package with:

    curl -O https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
    bash Miniconda3-latest-Linux-x86_64.sh

at some point it asks you if you wish to add conda to the path. You need to say yes there.

**Comment by gabrielrajkovic** [comment]
Yes, I did this

**Comment by ialbert** [comment]
I just got a note and confirmation that the latest conda does not work on windows.
Your problem is not related to this but you will need to reinstall anyway. So best to start there.

See the conda installation on Windows (I have just updated it) make sure to reload your page.

https://www.biostarhandbook.com/setup/setup-conda.html

In a nutshell you will need to remove your existing conda package with

    rm -rf miniconda3

then install a previous version of it like so:

    wget https://repo.continuum.io/miniconda/Miniconda3-4.3.21-Linux-x86_64.sh
    bash Miniconda3-4.3.21-Linux-x86_64.sh

Now if conda is not visible in your terminal it means you have not installed it properly.
